Makefile

```diff
@@ -173,7 +173,7 @@ TMP_DIR=$$(mktemp -d) ;\
 cd $$TMP_DIR ;\
 go mod init tmp ;\
 echo "Downloading $(2)" ;\
-GOBIN=$(PROJECT_DIR)/bin go get $(2) ;\
+GOBIN=$(PROJECT_DIR)/bin go install $(2) ;\
```

Comment:
I like where you're heading with this commit! However, this project is [currently pinned to go version 1.16](https://github.com/DopplerHQ/kubernetes-operator/blob/97af8737c219ab91a8e10b1b1373bb5e61fa2faf/.github/workflows/release.yml#L48) which doesn't support this installation syntax. I think my recommendation would be to move this commit to a different branch where we can upgrade go in the future.

Reply:
Per our discussion, upgraded the project to Go 1.20 to get us past the dependency hell we were hitting otherwise.